utils/cronjobs_osgeo_lxd/README.md

```diff
@@ -14,29 +14,29 @@ programmer's manual.
 * generate and deploy the GRASS GIS Web pages at <https://grass.osgeo.org/>:
   * `hugo_clean_and_update_job.sh`
 * GRASS GIS source code weekly snapshots:
-  * grass7-stable: `cron_grass7_relbranch_src_snapshot.sh`
-  * grass8-stable: `cron_grass8_relbranch_src_snapshot.sh`
-  * grass8-devel: `cron_grass8_main_src_snapshot.sh`
+  * grass-legacy: `cron_grass_legacy_relbranch_src_snapshot.sh`
```

Comment:
...while at it, do you want to use whole words for release branch? The branch names say releasebranch, although they are unfortunately not separated words, they are at least whole words.

Reply:
So, you mean like this?

* grass-legacy: `cron_grass_legacy_src_snapshot.sh`

etc.?

Reply:
My understanding was (implemented in [ac22e46](https://github.com/OSGeo/grass-addons/pull/814/commits/ac22e46618e967e41dbc79a9fc9371397de3e3b4) to [fb3fdbd](https://github.com/OSGeo/grass-addons/pull/814/commits/fb3fdbddcfed96ad1ec2b0d7194d51bc57a6b4cd)), like renaming from e.g. _cron_grass7_relbranch_src_snapshot.sh_ to _cron_grass_legacy_releasebranch_src_snapshot.sh_

As for the actual changes in the code: Fetching the version from github (https://raw.githubusercontent.com/osgeo/grass/main/include/VERSION) is tested locally. When reviewing changes, please note that the version is picked up from github at the time the script runs. So, depending on the order in which things are done during release, that may be correct (or not). I don`t know the details of the release procedure, but I am happy to adjust the code if needed.

Also special attention should be put on the patch version. Here only numbers are taken (e.g. _dev_ is removed). From the hardcoded example, that was my understanding how the patch version was defined...

Also I noticed that there is https://grass.osgeo.org/grass-stable/manuals/addons/, but no https://grass.osgeo.org/grass-legacy/manuals/addons/, yet https://grass.osgeo.org/grass7/manuals/addons/. So if we want to support a legacy version, that may be good to add on the webserver...

Reply:
> but no https://grass.osgeo.org/grass-legacy/manuals/addons/

I have now activated the `grass-legacy/` redirect.

Reply:
Good! The _README.md_ is adjusted accordingly. However, please note that the version numbers remain in the target log-files in the cron-table (_[cron_job_list_grass](https://github.com/OSGeo/grass-addons/pull/814/files#diff-00117571ec8d8ab7328a7bb3c1be723f0f8edf33be66e2758a58ea71052e2474)_). Ideas how to remove those are welcome...